src/relay/backend/vm/removed_unused_funcs.cc

```diff
@@ -76,6 +72,15 @@ struct CallTracer : ExprVisitor {
       }
       visiting_.insert(func);
       VisitExpr(func);
+    } else {
+      VisitExpr(expr);
+    }
+  }
+
+  void VisitExpr_(const CallNode* call_node) final {
+    CheckExpr(call_node->op);
+    for (auto param : call_node->args) {
+      CheckExpr(param);
     }
```

Comment:
Can the bug be fixed by adding the code here? Looks like the bug is we don't continue visit the expression if it's not FunctionNode or GlobalVarNode. 
````python
else {
      VisitExpr(expr);
    }
````

Reply:
Yes, it could be. I just moved it out to make it look more clear. How do you think?

Reply:
Works for me.